source/user-manual/wazuh-dashboard/troubleshooting.rst

```diff
@@ -219,6 +219,16 @@ If the restart does not solve the problem, we can execute this process manually:
 
 .. note:: These actions take into account that the index that stores the saved objects must have valid field mappings. The field mappings are defined through a template, so they should exist before the index is created. This template is added when Wazuh dashboard starts if it doesn’t exist.
 
+Application Not Found
+---------------------
+
+If after updating, when accessing the dashboard we find the message ``Application Not Found``, this may be because after the update the file ``/etc/wazuh-dashboard/opensearch_dashboards.yml`` has not been overwritten with the new changes. To fix this, please add the following line to the configuration file:
```

Comment:
```suggestion
If you encounter the message *Application Not Found* when accessing the Wazuh dashboard after upgrading, it might be that the configuration file ``/etc/wazuh-dashboard/opensearch_dashboards.yml`` wasn't overwritten with new changes. To resolve this issue, replace the following line in the configuration file:
```